lib/models/file_upload_settings.dart

```diff
@@ -11,7 +11,7 @@ class FileUploadSettings {
   final int? imageWebpQuality;
   final int imageMaxProcessingMP;
   final bool denyDoubleFileExtensions;
-  final List<dynamic> converterOptions;
+  final List<dynamic>? converterOptions;
```

Comment:
@PrimozRatej converterOptions is removed. So you can completely remove it from your code.